### PR TITLE
Added responsive Navbar

### LINKED ISSUE #70: Make responsive navbar
**Is your feature request related to a problem? Please describe.**
The current navbar in the website is kind of old age type, make it such that it look good 
edit the ```hotelontouch/src/components/NavbarComponent.js```

### PR DESCRIPTION
**Pull request by Muditxofficial**
Fixes #70 

#### Changes done

- [ ]

Screenshots of the changes (If any) -

Note: Please check Allow edits from maintainers. if you would like us to assist in the PR.

#### ✅️ By submitting this PR, I have verified the following

- [x] Worked on forked repo and starred the main repo
- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [ ] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]
we don't need a navlogo svg please give any suitable icon from react bootstrap

File: hotelontouch/public/navlogo.svg [hunk 1]
```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 16.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
```
Comment (ayan-biswas0412) [inline hotelontouch/public/navlogo.svg:2]: if you have not used this file please remove it

**Review by ayan-biswas0412 — CHANGES_REQUESTED** [review]

File: hotelontouch/public/navlogo.svg (hunk 1 above)
Comment (Muditxofficial) [reply hotelontouch/public/navlogo.svg:2]: Just wait for 10 minutes i am trying to rebase.

Comment (Muditxofficial) [reply hotelontouch/public/navlogo.svg]: Please review this.

File: hotelontouch/src/components/NavbarComponent.js [hunk 2]
```diff
@@ -1,21 +1,43 @@
...
+        </Form>
+      </Navbar.Collapse>
+    </Navbar>
+  );
 }
-
-export default NavbarComponent;
\ No newline at end of file
+export default NavbarComponent;
+library.add(faBuilding);
```
[the remark this answers is not in the document]
Comment (Muditxofficial) [reply hotelontouch/src/components/NavbarComponent.js:43]: it is used for icon in fontawesome library.

[the remark this answers is not in the document]
Comment (ayan-biswas0412) [reply hotelontouch/src/components/NavbarComponent.js:43]: is it needed to include here or there is other possibilities ?

**Review by ayan-biswas0412 — APPROVED** [review]
nice work

**Comment by Muditxofficial** [comment]
@ayan-biswas0412 please add labels to this pr and also see issue #25

**Comment by Muditxofficial** [comment]
@ayan-biswas0412 please add labels to this pr and also see issue #25

**Pull request merged**